src/documentation-helpers.ts

```diff
@@ -1048,7 +1049,7 @@ export const depictRequest = ({
   const bodyDepiction = excludeExamplesFromDepiction(
     excludeParamsFromDepiction(
       walkSchema({
-        schema: hasRaw(schema) ? ZodFile.create().buffer() : schema,
+        schema: hasRaw(schema) ? ez.file("buffer") : schema,
```

Comment:
Actually, with ez.raw() made propriatery, it can be placed into walker rules instead

Reply:
done in 181f3559